# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/tests/browser/text/browser.toml
# Commit: dbdf17d42860
# Full Hash: dbdf17d42860765bcf2e6aaab1b0cd71ad1565b9
# Author: James Teh <jteh@mozilla.com>
# Date: 2025-02-07 16:40:09
# Regressor Bug: 1909142
# File Overlap Count: 1
# Description:
#   Bug 1909142 part 2: Expose custom highlights via accessibility APIs. r=morgan
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D233806
# ==============================================================================

diff -r 9529f52eebf3 -r dbdf17d42860 accessible/tests/browser/text/browser.toml
--- a/accessible/tests/browser/text/browser.toml	Fri Feb 07 05:25:15 2025 +0200
+++ b/accessible/tests/browser/text/browser.toml	Fri Feb 07 02:03:56 2025 +0000
@@ -6,6 +6,7 @@
   "!/accessible/tests/mochitest/*.js",
 ]
 prefs = [
+  "dom.customHighlightAPI.enabled=true",
   "dom.text_fragments.enabled=true",
 ]
 